deployment/7_upgrade_1_6.js

```diff
@@ -157,6 +157,21 @@ const deploy = async (network) => {
       [wrapper.contractAddress, utils.asciiToBytes32(wrapper._contract.contractName)]);
   }
 
+  // ////////////////////////////////////////////////////
+  // Disable KyberNetwork on the TokenPriceProvider
+  // ////////////////////////////////////////////////////
+
+  if (!await TokenPriceProviderWrapper.contract.managers(deploymentWallet.address)) {
+    const TokenPriceProviderAddManagerTx = await TokenPriceProviderWrapper.contract.addManager(deploymentWallet.address, { gasPrice });
+    await TokenPriceProviderWrapper.verboseWaitForTransaction(TokenPriceProviderAddManagerTx,
+      `Set ${deploymentWallet.address} as the manager of the TokenPriceProvider`);
+  }
+
+  const TokenPriceProviderSetKyberNetworkTx = await TokenPriceProviderWrapper.contract.setKyberNetwork("0x0000000000000000000000000000000000000000",
+    { gasPrice });
+  await TokenPriceProviderWrapper.verboseWaitForTransaction(TokenPriceProviderSetKyberNetworkTx,
+    "Disable the KyberNetwork on the TokenPriceProvider");
+
```

Comment:
This was executed on the previous deployment and is no longer required. The address of kyber network is already set to `address(0)`.

Also, the backend addresses are already managers and the multisig is already the owner of the contract. Basically the next section (Change the owner of TokenPriceProvider) is no longer required either.